src/main/java/org/datavyu/views/VideoController.java

```diff
@@ -1606,31 +1606,36 @@ public void jogBackAction() {
             double frameRate = frameRateController.getFrameRate();
             long clockTime = (long) clockTimer.getClockTime();
             long stepSize = (long) Math.ceil(MILLI_IN_SEC / frameRate); // step size is in milliseconds
+            // Update the clock timer with the new time
+            long newTime = clockTime - (clockTime % stepSize) - stepSize;
             for (StreamViewer streamViewer : streamViewers) {
                 // TODO: Tie offset & duration to stream viewer only and pull it in the track model
                 TrackModel trackModel = tracksEditorController.getTrackModel(streamViewer.getIdentifier());
                 // We can only use the step function if this frame rate is close enough to the highest frame rate
                 if (streamViewer.isStepEnabled() && almostEqual(streamViewer.getFramesPerSecond(), frameRate,
                         ALMOST_EQUAL_FRAME_RATES)) {
                     streamViewer.stepBackward();
+                    logger.info("Jog Forward Stream " + streamViewer.getIdentifier() +
```

Comment:
Change log level: info -> debug

Rational: The user can continually press the step button which will clutter the log file

Reply:
Done!